### PR TITLE
PORTALS-2685: Small fixes, now with a No Content UI if no plots are d…

### PR DESCRIPTION
**Pull request by jay-hodgson**
…efined for the Featured Data Tabs component

### CODE REVIEW

File: apps/portals/src/configurations/elportal/resources.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   "SELECT * FROM syn51228429 WHERE `resourceType` = 'metadata'"
-export const peopleSql = 'SELECT * FROM syn51209684 ORDER BY firstName'
+export const peopleSql = 'SELECT * FROM syn51209684'
 export const projectsSql = 'SELECT * FROM syn51209786'
 export const publicationsSql = 'SELECT * FROM syn51209321'
 export const studiesSql = 'SELECT * FROM syn51209885 ORDER BY isFeatured DESC'
 export const speciesSql = 'SELECT * FROM syn51449110 ORDER BY ItemOrder'
 export const computationalSql = 'SELECT * FROM syn51469335'
-export const upsetPlotSql = 'SELECT individualID, assay FROM syn51581771'
+export const upsetPlotSql = 'SELECT files, assay FROM syn51581771'
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/elportal/resources.ts:10]: Schema changed

File: apps/portals/src/configurations/elportal/routesConfig.ts [hunk 2]
```diff
@@ -116,6 +116,15 @@ const routes: GenericRoute[] = [
                 ],
               },
             },
+            {
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/elportal/routesConfig.ts:119]: Add empty tab

File: apps/portals/src/configurations/elportal/routesConfig.ts [hunk 3]
```diff
@@ -128,7 +137,7 @@ const routes: GenericRoute[] = [
         props: {
           config: [
             {
-              title: 'Data Portals',
+              title: 'Data Repositories',
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/elportal/routesConfig.ts:140]: Copy changes

File: apps/portals/src/configurations/elportal/synapseConfigs/data.ts [hunk 4]
```diff
@@ -18,8 +18,7 @@ const data: SynapseConfig = {
           matchColumnName: 'study',
           isMarkdown: false,
           baseURL: 'Explore/Studies/DetailsPage',
-          URLColumnName: 'Study_Name',
-          wrapValueWithParens: true,
+          URLColumnName: 'studyKey',
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/elportal/synapseConfigs/data.ts:21]: Bad link, study page looks for "studyKey" search param

File: packages/synapse-react-client/src/components/FeaturedDataTabs/FeaturedDataTabs.tsx [hunk 5]
```diff
@@ -54,23 +56,30 @@ const FeaturedDataTabs: React.FunctionComponent<
...
-            <div className="bootstrap-4-backport FeaturedDataTabs__explore-all">
-              <Button
-                variant="secondary"
-                size="lg"
-                href={selectedTabProps.explorePagePath}
-              >
-                View All {selectedTabProps.exploreObjectType}
-              </Button>
-            </div>
+          {selectedTabProps.plotsConfig.configs.length > 0 ? (
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/components/FeaturedDataTabs/FeaturedDataTabs.tsx:59]: If there are no plot configurations, then show a NoContentAvailable UI (wrapped in a Paper)

**Review by nickgros — APPROVED** [review]

**Pull request merged**